src/test/java/lotto/LottosTest.java

```diff
@@ -0,0 +1,48 @@
+package lotto;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class LottosTest {
+
+    private Lottos lottos;
+    private Lotto lastWinningLotto;
+
+    private LottoResults lottoResults;
+
+    @BeforeEach
+    void createLottos() {
+        lottos = new Lottos(Arrays.asList(
+                new Lotto(Arrays.asList(1, 2, 3, 4, 5, 6)),
+                new Lotto(Arrays.asList(1, 2, 3, 7, 8, 9)),
+                new Lotto(Arrays.asList(7, 8, 9, 10, 11, 12))));
+    }
```

Comment:
```suggestion
    private Lottos lottos = new Lottos(Arrays.asList(
            new Lotto(Arrays.asList(1, 2, 3, 4, 5, 6)),
            new Lotto(Arrays.asList(1, 2, 3, 7, 8, 9)),
            new Lotto(Arrays.asList(7, 8, 9, 10, 11, 12)))
    );
    private Lotto lastWinningLotto = new Lotto(Arrays.asList(1, 2, 3, 4, 5, 6));
```
junit의 기본 라이프사이클은 테스트메소드당이기 때문에 바로 선언해주셔도 되어요.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle